tests/common/snappi_tests/common_helpers.py

```diff
@@ -821,6 +823,42 @@ def get_pfc_frame_count(duthost, port, priority, is_tx=False):
     return int(pause_frame_count.replace(',', ''))
 
 
+def get_tx_frame_count(duthost, port):
```

Comment:
unify these 2 functions - get_tx_frame_count and get_rx_frame_count

Reply:
unified in #9775. this PR is dependent on that PR.